### ISSUE TITLE
[question] Setup for nicotine plugins?

### ISSUE THREAD

**Issue by minicoz** [sirjmann92/nicotineplus-proper#12]
What is the docker volume setup to be able to add custom nicotine+ plugins? 

From your setup, it seems like its

/home/nicotine/.local/share/nicotine/plugins.

I keep getting

```
cp: can't stat '/home/nicotine/config-default': No such file or directory
```

**Comment by minicoz** [comment]
Plugins should be added to the directory

/home/Documents/Configs/NicotineTest/app/share/nicotine/plugins

for example I have these three

![image](https://github.com/user-attachments/assets/85428655-0d08-4ea5-869e-053244f79cd8)

**Comment by sirjmann92** [comment]
Interesting. I can create a symlink in the container for plugins, like I did for /config and /data, to make it a cleaner experience to map the plugins directory directly.

Thanks for the info!

I'll leave this open until I get it done and updated images uploaded.

**Comment by minicoz** [comment]
Yup I just tested it and now it works great. We have the plugins mapped under the config.

Thanks for your help!

**Comment by sirjmann92** [comment]
Thank you for confirming! Enjoy!

**Comment by sirjmann92** [comment]
@minicoz - just a heads up, I moved the plugins directory to /config/data/plugins. The directory will be created for you now, so you don't have to create it, but you will have to move your plugins.

Apologies for any inconvenience, but this made more sense. 

You'll see the changes the next time you pull a new image.
